### PR TITLE
docs(structural-directives): typos and clean-up

### PR DESCRIPTION
**Pull request by jsayol**

### CODE REVIEW

File: public/docs/ts/latest/guide/structural-directives.jade [hunk 1]
```diff
@@ -209,7 +209,7 @@ figure.image-display
 
 +makeExample('structural-directives/ts/app/structural-directives.component.html', 'ngFor-template')(format=".")
 :marked
-  The basic pattern is the same:&nbsp; create a `<template>`, relocate the content,
+  The basic pattern is the same: create a `<template>`, relocate the content,
```
Comment (wardbell) [inline public/docs/ts/latest/guide/structural-directives.jade]: Please revert to the `&nbsp;`.  I wanted that extra spacing.

File: public/docs/ts/latest/guide/structural-directives.jade [hunk 2]
```diff
@@ -5,13 +5,13 @@ block includes
...
   navigates, whole sections of the DOM appear and disappear according
-  to the application state. In this chapter we'll to look at how Angular
+  to the application state. In this chapter we'll look at how Angular
   manipulates the DOM and how we can do it ourselves in our own directives.
 
   In this chapter we will
   - [learn what structural directives are](#definition)
   - [study *ngIf*](#ngIf)
-  - [discover the &lt;template> element](#template)
+  - [discover the <template> element](#template)
```
Comment (wardbell) [inline public/docs/ts/latest/guide/structural-directives.jade]: Please revert the `&lt;` as our jade engine is confused by angle brackets.

**Comment by jsayol** [comment]
Makes sense. Reverted 👍

**Comment by wardbell** [comment]
Thanks!

**Pull request merged**